app/views/passengers/show.haml

```diff
@@ -0,0 +1,39 @@
+%p#notice= notice
+
+%h1= @passenger.name
```

Comment:
:+1: for the convention of having your page 'title' be the name of the object rather than just 'Displaying Passenger' or something painfully obvious and unhelpful.